docs/reference/query-dsl/intervals-query.asciidoc

```diff
@@ -125,51 +165,112 @@ WARNING: Avoid beginning patterns with `*` or `?`. This can increase
 the iterations needed to find matching terms and slow search performance.
 --
 `analyzer`::
-Which analyzer should be used to normalize the `pattern`.  By default, the
-search analyzer of the top-level field will be used.
+(Optional, string) <<analysis, analyzer>> used to normalize the `pattern`.
+Defaults to the top-level `<field>`'s analyzer.
+
 `use_field`::
-If specified, then match intervals from this field rather than the top-level field.
-The `pattern` will be normalized using the search analyzer from this field, unless
++
+--
+(Optional, string) If specified, match intervals from this field rather than the
+top-level `<field>`.
+
+The `pattern` is normalized using the search analyzer from this field, unless
 `analyzer` is specified separately.
+--
 
 [[intervals-all_of]]
-==== `all_of`
+==== `all_of` rule parameters
 
-`all_of` returns returns matches that span a combination of other rules.
+The `all_of` rule returns matches that span a combination of other rules.
 
-[horizontal]
 `intervals`::
-An array of rules to combine.  All rules must produce a match in a
-document for the overall source to match.
+(Required, array of rule objects) An array of rules to combine. All rules must
+produce a match in a document for the overall source to match.
+
 `max_gaps`::
-Specify a maximum number of gaps between the rules.  Combinations that match
-across a distance greater than this will not match.  If set to -1 or
-unspecified, there is no restriction on this distance.  If set to 0, then the
-matches produced by the rules must all appear immediately next to each other.
++
+--
+(Optional, integer) Maximum number of positions between the matching terms.
```

Comment:
s/terms/intervals produced by the rules/

Reply:
Fixed with fe1a4a6a4dd.